src/helpers/utilities.js

```diff
@@ -161,12 +162,12 @@ export function getItemTypesThatCanStack() {
 						return [key, schema.fields ?? true]
 					})
 				itemTemplate.system = Object.fromEntries(itemTemplate.system);
-			} else if (game.system?.template?.Item?.[type]) {
-				itemTemplate.system = foundry.utils.deepClone(game.system.template.Item[type]);
-				if (itemTemplate.system?.templates?.length) {
-					const templates = foundry.utils.duplicate(itemTemplate.system.templates);
+			} else if (game.system?.documentTypes?.Item?.[type]) {

```

Comment:
This won't work in v11, as `game.system?.documentTypes.Item` is an array, and this is expecting it to be an object. Systems that aren't using the DataModel workflow will most likely use the traditional `game.system.template` workflow, so this is not something that should change. I'm going to approve this PR, but change this in the final release.